lookout/values.yaml

```diff
@@ -71,12 +59,13 @@ resources:
 
 nodeSelector:
   lookout: {}
-  dummyAnalyzer: {}
 
 tolerations:
   lookout: []
-  dummyAnalyzer: []
 
 affinity:
   lookout: {}
-  dummyAnalyzer: {}
+
+rabbitmq-ha: 
```

Comment:
I'd add documentation to clarify this is where you pass options to rabbit

Reply:
Okay!